src/main/resources/templates/snowflake/shared/create-table-evolve-schema.ssp

```diff
@@ -8,9 +8,15 @@
 }#
 
 #if (tableDiff.existsInDestination && tableDiff.existsInSource)
-    #if (tableDiff.allChangesAreCompatible())
-        ALTER TABLE ${destination.stagingDatabase.name}.${destination.stagingDatabase.schema}.${tableDiff.destinationName} ADD COLUMN ${unescape(tableDiff.columnDDL(typeMapping))};
-    #elseif (!tableDiff.allChangesAreCompatible())
+    #if (tableDiff.allChangesAreCompatible(typeMapping))
+        #if (tableDiff.isColumnAdded())
+            ALTER TABLE ${destination.stagingDatabase.name}.${destination.stagingDatabase.schema}.${tableDiff.destinationName} ADD COLUMN ${unescape(tableDiff.columnDDL(typeMapping))};
+        #end
+        #if (tableDiff.isColumnSizeChanged(typeMapping))
+            ALTER TABLE ${destination.stagingDatabase.name}.${destination.stagingDatabase.schema}.${tableDiff.destinationName} ALTER ${unescape(tableDiff.alterColumnDDL(typeMapping))};
+        #end
```

Comment:
I think there will be a problem if a table has two changes, one is a column add and one is a column size extension. In that case I think we will do both alters for both columns.

I am thinking isColumnAdded should be "getColumnAdds" and then we iterate over the list. Similar with isColumnSIzeChanged.

Reply:
I think this scenario is already taken care. If a column is added we are using [columnDDL](https://github.com/phdata/streamliner/blob/master/src/main/java/io/phdata/streamliner/schemadefiner/model/TableDiff.java#L45) where getIsAdd filter is added. And for column size extension, [alterColumnDDL](https://github.com/phdata/streamliner/pull/64/commits/f1a33468150502d8505f681cff62d0e29f0b7d14#diff-aa3a08ed84eb7ff3bf908edc74c6c83755c9d888802587d0a4cf6ec2aaf9f67bR137) where isColumnSizeChanged filter is added. 
Even in the Junit test case the same scenario is tested.

Reply:
Ok gotcha, great.